nemo/collections/tts/modules/radtts.py

```diff
@@ -812,8 +878,18 @@ def forward_for_export(self, text, lens, speaker_id, speaker_id_text, speaker_id
             sigma_txt=0.0,
             sigma_f0=1.0,
             sigma_energy=1.0,
-            f0_mean=145.0,
-            f0_std=30.0,
+            f0_mean=0.0,
+            f0_std=0.0,
```

Comment:
Those mean values should be read from the checkpoint.

Reply:
A value of 0 means no shift with is what we want.